ert/data/_record.py

```diff
@@ -172,11 +176,15 @@ def ensemble_size_validator(
         return ensemble_size
 
     @root_validator(skip_on_failure=True)
-    def ensure_consistent_ensemble_size(
+    def ensure_consistent_ensemble(
         cls, ensemble_record: Dict[str, Any]
     ) -> Dict[str, Any]:
         assert "records" in ensemble_record and "ensemble_size" in ensemble_record
         assert len(ensemble_record["records"]) == ensemble_record["ensemble_size"]
+        record_type = ensemble_record["records"][0].record_type
```

Comment:
Should we add a check that `ensemble_size > 0` in the ensemble size validator (and also a test for it)? I dont think we allow EnsembleRecords without any data (or at least it makes no sense to talk about the data type in that case), so we should probably just fail the validation :slightly_smiling_face:

Reply:
Indeed, done.